pyproject.toml

```diff
@@ -7,19 +7,12 @@ line_length = 100
 
 [tool.black]
 line-length = 100
-include = '\.pyi?$'
-exclude ='''
+include = '''
 /(
-		  \.eggs	# exclude a few common directories
-		| \.git		# in the root of the project
-		| \.hg
-		| \.mypy_cache
-		| \.nox
-		| \.tox
-		| \.venv
-		| _build
-		| buck-out
-		| build
-		| dist
+      \.pyi?$
+    | setup.py
+    | src
+    | test
+    | examples
```

Comment:
This looks like it'll run on any file with `src` anywhere in its filename, even if it doesn't have the right extension, or isn't part of the top-level `src` tree.

What happens if you just remove this setting entirely (the default is `(\.pyi?|\.ipynb)$`)? According to the docs, if you don't specify an `exclude` then it will automatically honour `.gitignore`, which should prevent it reformatting anything that isn't part of the source.

Reply:
That seems to have worked.

I think if I merge, I'm going to do a rebase and squash everything into a single commit, so that I can add a little file saying which commit to ignore for git blame:
https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame